src/components/CohortBuilder/Summary/index.js

```diff
@@ -39,118 +39,95 @@ const CardSlot = styled(Card)`
-              sortBy={sortDescParticipant}
-              tickInterval={4}
-              colors={[theme.chartColors.blue, theme.chartColors.purple]}
-              xTickTextLength={28}
-              legends={[
-                { title: 'Probands', color: theme.chartColors.blue },
-                { title: 'Family Members', color: theme.chartColors.purple },
-              ]}
-              padding={0.5}
-            />
-          </CardSlot>
-        </Col>
-        <Col sm={12} md={md} lg={lg}>
-          <CardSlot title="Most Frequent Diagnoses">
-            <HorizontalBar
-              style={{ maxWidth: '100px' }}
-              data={topDiagnosesBarMock}
-              indexBy="label"
-              keys={['probands', 'familyMembers']}
-              tooltipFormatter={mostFrequentDiagnosisTooltip}
-              sortByValue={true}
-              tickInterval={4}
-              colors={[theme.chartColors.blue, theme.chartColors.purple]}
-              xTickTextLength={28}
-              legends={[
-                { title: 'Probands', color: theme.chartColors.blue },
-                { title: 'Family Members', color: theme.chartColors.purple },
-              ]}
-              padding={0.5}
-            />
-          </CardSlot>
-        </Col>
-        <Col sm={12} md={md} lg={lg}>
-          <CardSlotPies>
-            <Pie
-              style={{ height: '42%', width: '50%', marginBottom: '10px', marginTop: '5px' }}
-              title={'Gender'}
-              data={demographicPiesMock.gender}
-              colors={[theme.chartColors.orange, '#FFFFFF']}
-            />
-            <Pie
-              style={{ height: '42%', width: '50%', marginBottom: '10px', marginTop: '5px' }}
-              title={'Ethnicity'}
-              data={demographicPiesMock.ethnicity}
-              colors={[theme.chartColors.darkblue, '#FFFFFF']}
-            />
-            <Pie
-              style={{ height: '42%', width: '50%' }}
-              title={'Race'}
-              data={demographicPiesMock.race}
-              colors={[theme.chartColors.lightpurple, '#FFFFFF']}
-            />
-            <Pie
-              style={{ height: '42%', width: '50%' }}
-              title={'Family Composition'}
-              data={demographicPiesMock.familyComposition}
-              colors={[theme.chartColors.lightblue, '#FFFFFF']}
-            />
-          </CardSlotPies>
-        </Col>
-        <Col sm={12} md={md} lg={lg}>
-          <CardSlot title="File Breakdown" />
-        </Col>
-        <Col sm={12} md={md} lg={lg}>
-          <CardSlot title="Age at Diagnosis" />
-        </Col>
-      </Row>
-    </Col>
-    <Col sm={12} md={3} lg={3}>
-      <LongCard title="Phenotypes">
-        <pre>{JSON.stringify(sqon, null, 2)}</pre>
-      </LongCard>
-    </Col>
-  </Row>
+const Summary = ({ theme, sqon, api }) => (
+  <QueriesResolver api={api} sqon={sqon} queries={[demographicQuery({ ...{ sqon } })]}>
```

Comment:
@ciaranschutte I don't think this needs to be another HOC, but that is just my preference.

Reply:
@nyanofthemoon After talking with @hlminh2000 HOC is probably best for now.
So we have a state provider for the SQONs to the page
We also need to provide data to the charts - this is what the `QueriesResolver` is for

Reply:
The state provider only provides the sqon, `QueriesResolver` resolves that sqon into actual data from the API.

Reply:
@jberube yup, I mentioned a solution to that problem below :) the implementation of `QueriesResolver` can change but you got the idea ;)